MonoGame.Framework.Content.Pipeline/Audio/AudioContent.cs

```diff
@@ -281,6 +281,10 @@ public void ConvertFormat(ConversionFormat formatType, ConversionQuality quality
                     channelCount,
                     format,
                     sampleRate);
+
+                // Loop start and length in number of samples. Defaults to entire sound
+                loopStart = 0;
+                loopLength = data.Count / ((bitsPerSample / 8) * channelCount);
```

Comment:
Then what was `loopLength /= bytesPerSample` in the runtime side doing?  Was `loopLength` 0 there too?